lib/engine/step/g_18_al/train.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
```

Comment:
Looks good, only real comment is the use of 'g_18_al' when it's common to nearly all of Mark Derrick's games.
How about g_18_markderrick?

or train_buyone_until_phase4.rb?

Reply:
Renamed it to a descriptive name, and put the file in lib/engine/step.